src/lava/magma/runtime/runtime_service.py

```diff
@@ -16,6 +16,33 @@
     MGMT_COMMAND,
 )
 
+"""This file defines the interface for RuntimeService which is responsible for
```

Comment:
'defines the AbstractRuntimeService' interface or 'interface for a RuntimeService'